server/src/routes/api/v1/auth/index.ts

```diff
@@ -0,0 +1,41 @@
+import { Router, Request, Response, NextFunction } from "express";
+import { AuthService } from "./../../../../logic/services/auth.service";
+import { ErrorUtils } from "./../../../../logic/utils/errorUtils";
+import { ApiError, DbError, SocoboUser } from "./../../../../models/index";
+
+
+export class AuthRouteV1 {
+  constructor (private _authService: AuthService, private _router: Router) {}
+
+  createRoutes (): Router {
+    // login the user
+    this._router.post("/login", (req: Request, res: Response, next: NextFunction) => {
+      this._authService.login(req.body.email, req.body.password)
+        .then((result: SocoboUser) => res.status(200).json(result))
+        .catch((error: any) => {
+          if (ErrorUtils.notFound(error)) {
+            res.status(404).json(
+                new DbError(`The user login is failed - ${error.message}!`, AuthService.name, 
```

Comment:
Maybe the error message should be a bit more descriptive. Does this mean that the user for the login is not registered yet?